docs/GlobalAPI.md

```diff
@@ -59,7 +59,13 @@ If you want to run some cleanup after every test instead of after all tests, use
 
 Runs a function after each one of the tests in this file completes. If the function returns a promise or is a generator, Jest waits for that promise to resolve before continuing.
 
-Optionally, you can provide a `timeout` (in milliseconds) for specifying how long to wait before aborting. _Note: The default timeout is 5 seconds._
+Optionally, you can provide a `timeout` (in milliseconds) for specifying how long to wait before aborting.
+
+:::note
+
+The default timeout is 5 seconds.
+
+:::
```

Comment:
Perhaps all should be reworked for consistency?

Reply:
they should 👍

Reply:
@Biki-das could you update this?

Reply:
Yeah sure caught up in something will do it soon